src/libexpr/eval.hh

```diff
@@ -65,8 +65,12 @@ std::string printValue(const EvalState & state, const Value & v);
 std::ostream & operator << (std::ostream & os, const ValueType t);
 
 
-// FIXME: maybe change this to an std::variant<SourcePath, URL>.
-typedef std::pair<std::string, std::string> SearchPathElem;
+struct SearchPathElem
+{
+    std::string prefix;
+    // FIXME: maybe change this to an std::variant<SourcePath, URL>.
+    std::string path;
```

Comment:
These names are the same names used by `builtins.getContext`. I think that's fine for now. They are not the best names but it is good to be consistent.

I eventually want this to be a `std::variant` to better distinguish the `=` and no-`=` cases, and then we can improve the names (and add API docs).